docker/Dockerfile

```diff
@@ -22,7 +23,8 @@ ENV PATH=/opt/py3/bin:$PATH
 
 RUN python3 -m pip install --no-cache-dir --upgrade pip setuptools==69.5.1 &&\
     python3 -m pip install --no-cache-dir torch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} --index-url https://download.pytorch.org/whl/${CUDA_VERSION_SHORT} &&\
-    python3 -m pip install --no-cache-dir cmake packaging wheel
+    python3 -m pip install --no-cache-dir cmake packaging wheel &&\
+    rm -rf ~/.cache
```

Comment:
what's the motivation of `rm -rf ~/.cache`?

Reply:
> what's the motivation of `rm -rf ~/.cache`?
for remove the pip cache and reduce image size;